doc/sflow/sflow_hld.md

```diff
@@ -786,6 +852,75 @@ sai_create_hostif_table_entry_fn(&host_table_entry, 4, sai_host_table_attr);
 
 It is assumed that the trap group and the trap itself have been defined using sai_create_hostif_trap_group_fn() and sai_create_hostif_trap_fn().
 
+### 6.10 Mapping dropped packet traps to a GENETLINK host interface multicast group
+
+Below is an example code snip that shows how a GENETLINK based host inerface is created.
+
+```
+sai_object_id_t hostif_id;
+sai_attribute_t hostif_attr[3];
+
+hostif_attr[0].id=SAI_HOSTIF_ATTR_TYPE;
+hostif_attr[0].value=SAI_HOSTIF_TYPE_GENETLINK;
+
+hostif_attr[1].id= SAI_HOSTIF_ATTR_NAME;
+hostif_attr[1].value="NET_DM";
+
+sai_host_if_attr[2].id= SAI_HOSTIF_ATTR_GENETLINK_MCGRP_NAME;
+sai_host_if_attr[2].value="packets";
+
+aai_create_hostif_fn(&hostif_id, 3, hostif_attr);
+```
+
+Below is the code snip that outlines how a dropped packet notifications are mapped to the GENETLINK host interface.
+
+```
+sai_object_id_t table_entry;
+sai_attribute_t table_attr[4];
+
+table_attr[0].id=SAI_HOSTIF_TABLE_ENTRY_ATTR_TYPE;
+table_attr[0].value=SAI_HOSTIF_TABLE_ENTRY_TYPE_TRAP_ID;
+
+table_attr[1].id=SAI_HOSTIF_TABLE_ENTRY_ATTR_TRAP_ID;
+table_attr[1].value=discard_packet_trap_id; // Object referencing DISCARD packet traps
+
+table_attr[2].id=SAI_HOSTIF_TABLE_ENTRY_ATTR_CHANNEL;
+table_attr[2].value=SAI_HOSTIF_TABLE_ENTRY_CHANNEL_TYPE_GENETLINK;

```

Comment:
@rck-innovium shouldn't be used "events" as MCGRP NAME for drop notifications? 
sai_host_if_attr[2].id= SAI_HOSTIF_ATTR_GENETLINK_MCGRP_NAME;
**sai_host_if_attr[2].value="events";**

Reply:
Yes, you are right.

```c

    attr_list[attr_count].id = SAI_HOSTIF_ATTR_GENETLINK_MCGRP_NAME;
    strncpy(attr_list[attr_count].value.chardata, "events", 31);
    attr_list[attr_count].value.chardata[31] = '\0';
    attr_count++;
```